core/field_dropdown.ts

```diff
@@ -741,16 +652,90 @@ const IMAGE_Y_OFFSET = 5;
 /** The total vertical padding above and below an image. */
 const IMAGE_Y_PADDING: number = IMAGE_Y_OFFSET * 2;
 
-/** Android can't (in 2014) display "▾", so use "▼" instead. */
-FieldDropdown.ARROW_CHAR = userAgent.ANDROID ? '▼' : '▾';
+/**
+ * NOTE: Because Sentinel is an empty class, proving a value is Sentinel does
+ * not resolve in TS that it isn't a MenuGenerator.
+ */
+function isMenuGenerator(menuGenerator: MenuGenerator|
+                         Sentinel): menuGenerator is MenuGenerator {
+  return menuGenerator !== Field.SKIP_SETUP;
+}
+
+/**
+ * Factor out common words in statically defined options.
+ * Create prefix and/or suffix labels.
+ */
+function trimOptions(options: MenuOption[]):
+    {options: MenuOption[]; prefix?: string; suffix?: string;} {
```

Comment:
I reeeeally like this change :D

Reply:
I moved the comment to right above the return statement.

Glad you like the change! ⭐